boards/derfmega128/include/board.h

```diff
@@ -30,9 +30,8 @@ extern "C" {
  * @{
  */
 #define XTIMER_WIDTH        (16)
-#define XTIMER_HZ           (CLOCK_CORECLOCK / 64)
+#define XTIMER_HZ           (250000UL)
```

Comment:
Can you explain why this change?

Reply:
It was raising the preprocessor error ""XTIMER_SHIFT cannot be derived for given XTIMER_HZ, verify settings!"". So I guess the expression was too complex for the preprocessor to correctly take into account the divider expression. I also tried `CLOCK_CORECLOCK >> 6` but it didn't work either.
Since CLOCK_CORECLOCK is 16MHZ, the resulting value is 250kHz anyway.

But I'm opened to better suggestions of course :)

Reply:
When did this show up? What applicaiton?

Reply:
(trying to reproduce :) )

Reply:
I couldn't reproduce either but I'm pretty sure it was failing on Murdock. Let's see how it goes there again.

Reply:
So it seems that this change is no longer needed: Murdock is green now (after a rebase). I'm wondering what has changed in the meantime.